src/common/runtime/wpt.ts

```diff
@@ -17,19 +22,60 @@ declare function setup(properties: { explicit_done?: boolean }): void;
 declare function promise_test(f: (t: WptTestObject) => Promise<void>, name: string): void;
 declare function done(): void;
 
+declare let loadWebGPUExpectations: Promise<RawTestQueryStringWithExpectation[]> | undefined;
```

Comment:
Perhaps add a line in the default `cts.html` template that sets this to `undefined` explicitly, so there's a reference for how to use it.

Reply:
Done.